packages/e2e/cypress/fixtures/force-change-password.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ChallengeName": "NEW_PASSWORD_REQUIRED",
```

Comment:
@slaymance I went to this user pool, created a test user for this state, and gave them a temporary password.

Is this the same as `FORCE_CHANGE_PASSWORD`?